constants/constants.go

```diff
@@ -0,0 +1,31 @@
+package constants
+
+import (
+	"errors"
+	"path/filepath"
+)
+
+const (
+	// WorkspacesDir is the path to the directory where
+	// all workspaces are stored by default.
+	WorkspacesDir = "/workspaces"
+
+	// EmptyWorkspaceDir is the path to a workspace that has
+	// nothing going on... it's empty!
+	EmptyWorkspaceDir = WorkspacesDir + "/empty"
+
+	// MagicDir is where all envbuilder related files are stored.
+	// This is a special directory that must not be modified
+	// by the user or images.
+	MagicDir = "/.envbuilder"
+)
+
+var (
+	ErrNoFallbackImage = errors.New("no fallback image has been specified")
+
+	// MagicFile is a file that is created in the workspace
+	// when envbuilder has already been run. This is used
+	// to skip building when a container is restarting.
+	// e.g. docker stop -> docker start
+	MagicFile = filepath.Join(MagicDir, "built")
+)
```

Comment:
I do feel like this and options belong in the same package, how about merging them under `core` or something? Not sure it's better description wise but still. Ideally I'd love for these and options to be in the package named `envbuilder` since that makes the most sense. But then we either lose the separation, or need to move the Run method elsewhere.

I suppose a `build` package could be appropriate for those, but then we'd kinda revert some of the splits you've already been doing.